content/source/docs/cloud/getting-started/workspaces.html.md

```diff
@@ -34,30 +34,13 @@ To create your first workspace, click the "+ New Workspace" button in the upper
 
 ![new button](./images/work-new-button.png)
 
-On the "Create a new Workspace" page, you need to enter at least two items: a workspace name, and a VCS repository. (You can search for a repo by typing part of its name.) When you've finished, click the "Create Workspace" button.
```

Comment:
The getting started section of the Cloud documentation looks very much like the getting stated Cloud learn guides, and they don't feel very docs-y to me. Do you think it would be possible to link the whole section out to learn? It hurts my heart to see duplicated content, and all the sections here seem more completely and clearly documented in later sections. 

I know this is kind of a philosophical question, so don't let it hold up this PR.

Reply:
- Yes, let's
- Not right at this moment tho 

That's another ticket, and is already something I want to do! But this needed a quick band-aid today because I was moving something it linked to.